deployment/README.md

```diff
@@ -0,0 +1,43 @@
+## Example deployment using docker-compose
+
+This is an example of a deployment of `vote` using docker-compose.
+
+## Start service
+
+```bash
+$ docker-compose up -d
+```
+
+## Stop the service, and delete all the data
+
+```bash
+$ docker-compose down
+```
+
+### Create an admin-user
+
+```bash
+# Replace <username> with the admin-username, and <unique-value> with a random value
+$ docker-compose exec vote ./bin/users create-admin <username> <unique-value>
```

Comment:
`unique-value` should be renamed to cardKey as that's what gets inserted into Mongo. However it's impossible to know the cardKey for the admin card without actually scanning it first.

Reply:
Yee. This is just for simplicity

Imo we can just remove the cardKey for admin-users. You will never need to deactivate an admin-user, or atleast I can never find any reason for it. There should only exist one admin-user, and deactivating the only user makes no sense to me